WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -149,14 +149,14 @@ await foreach (var offendingAlices in CheckTxoSpendStatusAsync(round, cancel).Co
 					try
 					{
 						int banCounter = 0;
-						var aliceDictionary = round.Alices.Where(x => !x.IsPayingZeroCoordinationFee).ToDictionary(a => a.Coin.ScriptPubKey, a => a);
+						var aliceDictionary = round.Alices.Where(x => !x.IsPayingZeroCoordinationFee).ToDictionary(a => a.Coin.Outpoint, a => a);
```

Comment:
```suggestion
						var aliceDictionary = round.Alices.Where(x => !x.IsPayingZeroCoordinationFee).ToDictionary(a => a.Coin, a => a);
```

Reply:
I will commit this, but what's wrong with the `Outpoint`? Unnecessary and `Coin` is more than enough?

Reply:
Nothing is wrong with `outpoint`, it is just unnecessary, that's all.